### ISSUE TITLE
Javascript Excel addin is not working for some users after upgradation of office 365

### ISSUE THREAD

**Issue by sureshbrp** [OfficeDev/office-js#666]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

I traced this issue in fiddler and i came to know that the below error
![image](https://user-images.githubusercontent.com/8668231/62869257-79e38f80-bce5-11e9-908b-1cf088ea4c61.png)

**Comment by RuoyingLiang** [comment]
Could you help to provide below information: 

- Which add-in on what platform ( e.g. Windows, Online, etc)? 
- What's the Office version number? 
- What's the behavior of the add-in? Is there error on status bar?

**Comment by sureshbrp** [comment]
We have developed a customized javascript excel addin to post the data from excel to mvc application grid and it is on the **windows platform**. 
We designed this addin installation/addition to the excel from the manifest file of shared folder. 
We are unable to get the response from addin UI, when we click on addin customized button after upgradation of Office 365.  So, I traced in Fiddler and got the 404 error as i told in my previous comment.
Currently we are using **Office 2016 with the Version 1902 (Build 11328.20368 Clik-to-Run)**
